assayist/processor/loose_artifact_analyzer.py

```diff
@@ -24,6 +24,12 @@ class LooseArtifactAnalyzer(Analyzer):
     and then look them up in koji by checksum. You can accomplish the same thing with
     koji-build-finder, but for our use-case doing it outselves was just as easy and made
     dependency management and debugging a lot simpler.
+
+    The question on what to do with artifacts found embedded in the source dir is a good one.
+    The schema does not currently alow for embedding artifacts in a SourceLocation. Instead
+    the most reasonable (and safe) thing is that is consistent with our existing schema /
+    queries is to assume that artifacts found embedded in the source are in fact embedded
```

Comment:
Perhaps we should note that the assumption is that embedded artifacts found in source packages (whether maven or rpm) will be connecting with their respective artifacts so any searches for "stuff embedded in the source" should simply filter out the source artifacts of a build and look what is embedded in them. Or at least that's my understanding of this comment.

Reply:
If I'm understanding you correctly I don't think that's what I'm saying. So if `python.rpm` is directly embedded in the source of a build, and the build produces `a.noarch.rpm` and `b.noarch.rpm` and `a.src.rpm`, there will be a connection between the `python.rpm` and all three build artifacts.